crates/bifrost/src/providers/replicated_loglet/sequencer/mod.rs

```diff
@@ -108,6 +147,8 @@ impl<T: TransportConnect> Sequencer<T> {
         record_cache: RecordCache,
         global_tail: TailOffsetWatch,
     ) -> Self {
+        describe_metrics();
```

Comment:
metrics will be initialized in bifrost if you move the keys to replicated_loglet/metric_definitions.rs